SPECS/munge/munge.spec

```diff
@@ -61,12 +57,11 @@ echo "d /run/munge 0755 munge munge -" > src/etc/munge.tmpfiles.conf.in
 # Get rid of some rpaths for /usr/sbin
 sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
 sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
-make %{?_smp_mflags} 
+make %{?_smp_mflags}
 
 
 %install
 
-rm -rf %{buildroot}
 make install DESTDIR=%{buildroot}
```

Comment:
Similar suggestion: let's update this line to `%make_install`.

Reply:
updated.